src/containers/blocks.jsx

```diff
@@ -95,14 +95,24 @@ class Blocks extends React.Component {
         this.ScratchBlocks.statusButtonCallback = this.handleConnectionModalStart;
         this.ScratchBlocks.recordSoundCallback = this.handleOpenSoundRecorder;
 
-        this.ScratchBlocks.FieldColourSlider.activateEyedropper_ = this.props.onActivateColorPicker;
+        // this.ScratchBlocks.FieldColourSlider.activateEyedropper_ = this.props.onActivateColorPicker;
         this.ScratchBlocks.Procedures.externalProcedureDefCallback = this.props.onActivateCustomProcedures;
         this.ScratchBlocks.ScratchMsgs.setLocale(this.props.locale);
 
+        const theme = this.ScratchBlocks.Theme.defineTheme('Scratch', {
+           'base': this.ScratchBlocks.Themes.Zelos,
+           'startHats': true
+        });
         const workspaceConfig = defaultsDeep({},
             Blocks.defaultOptions,
             this.props.options,
-            {rtl: this.props.isRtl, toolbox: this.props.toolboxXML, colours: getColorsForTheme(this.props.theme)}
+            {
+                rtl: this.props.isRtl,
+                toolbox: this.props.toolboxXML,
+                colours: getColorsForTheme(this.props.theme),
```

Comment:
Ok I am very confused about what the actual point of this is. It looks like we're defining a theme (line 102 above), which we're then using to [overwrite a theme](https://github.com/gonfunko/scratch-gui/blob/modern-blockly/src/lib/themes/default/index.js)? Which we're then using to assign colors to properties on the block, which get assigned to the SVG... somewhere.

Unless I'm misunderstanding, it feels like we're going through a lot of extra steps when we could just use the theme system. Does that match your understanding of what's happening? If so, can you file a bug for us to clean this up (or at least to consider doing so)?

Reply:
Yeah, I'm just trying to use Zelos so that it looks like Scratch. Filed #3 to track cleaning up the rest.